.github/workflows/linux-tests.yml

```diff
@@ -0,0 +1,125 @@
+name: GitHub Actions CI - (Linux)
```

Comment:
```suggestion
name: GitHub Actions CI (Linux)
```

Reply:
I think it'd be good for us to figure out a way to unify this with `tests.yml`.

Reply:
Done (changed the name)